src/components/Greeting.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components/macro";
+
+export default function Greeting({greeting, username}){
+
+	let name = username[0].toUpperCase() + username.slice(1)
+	name = name.substring(0, name.indexOf("."))
```

Comment:
Maybe add that to a separate function? 
```suggestion
function getName(username) {
	const name = username[0].toUpperCase() + username.slice(1)
	return name.substring(0, name.indexOf("."))
}

const name = getName(username)
```

Don't know why, but for me that looks a bit neater. But as usual, that is just my opinion!
(you could even extract that function and write a little unit test for it! 😃